src/__test__/NotFoundCountry.test.js

```diff
@@ -0,0 +1,20 @@
+// test this component
+import React from 'react';
+import renderer from 'react-test-renderer';
+import { Provider } from 'react-redux';
+import { BrowserRouter } from 'react-router-dom';
+import store from '../redux/store/index';
+import NotFoundCountry from '../Components/NotFoundCountry';
+
+describe('NotFoundCountry', () => {
+  it('renders NotFoundCountry component', () => {
+    const m = renderer.create(
+      <Provider store={store}>
+        <BrowserRouter>
+          <NotFoundCountry />
+        </BrowserRouter>
+      </Provider>,
+    ).toJSON();
+    expect(m).toMatchSnapshot();
+  });
+});
```

Comment:
- [ ] When I ran the  `npm test` command, I noticed that not all the tests passed. KIndly ensure that your tests pass as it gives confidence that your application works properly. 👍 😎 

![image](https://user-images.githubusercontent.com/77106962/220476259-06a4fc87-0673-44e5-893b-5b034aa2d4b3.png)